x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/trial_license_complete_tier/execution_logic/machine_learning_alert_suppression.ts

```diff
@@ -0,0 +1,1104 @@
+ * 2.0.
+ */
+
+import { expect } from 'expect';
+
+import {
+  MachineLearningRuleCreateProps,
+  RuleExecutionStatusEnum,
+} from '@kbn/security-solution-plugin/common/api/detection_engine';
+import type { Anomaly } from '@kbn/security-solution-plugin/server/lib/machine_learning';
+import {
+  ALERT_LAST_DETECTED,
+  ALERT_START,
+  ALERT_SUPPRESSION_DOCS_COUNT,
+  ALERT_SUPPRESSION_END,
+  ALERT_SUPPRESSION_START,
+  ALERT_SUPPRESSION_TERMS,
+  TIMESTAMP,
+} from '@kbn/rule-data-utils';
+import { ALERT_ORIGINAL_TIME } from '@kbn/security-solution-plugin/common/field_maps/field_names';
+import { DETECTION_ENGINE_SIGNALS_STATUS_URL as DETECTION_ENGINE_ALERTS_STATUS_URL } from '@kbn/security-solution-plugin/common/constants';
+import { EsArchivePathBuilder } from '../../../../../../es_archive_path_builder';
+import { FtrProviderContext } from '../../../../../../ftr_provider_context';
+import {
+  dataGeneratorFactory,
+  executeSetupModuleRequest,
+  forceStartDatafeeds,
+  getAlerts,
+  getOpenAlerts,
+  getPreviewAlerts,
+  patchRule,
+  previewRule,
+  previewRuleWithExceptionEntries,
+  setAlertStatus,
+} from '../../../../utils';
+import {
+  createRule,
+  deleteAllAlerts,
+  deleteAllAnomalies,
+  deleteAllRules,
+} from '../../../../../../../common/utils/security_solution';
+import { deleteAllExceptions } from '../../../../../lists_and_exception_lists/utils';
+
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+  const es = getService('es');
+  const log = getService('log');
+  const config = getService('config');
+
+  const isServerless = config.get('serverless');
+  const dataPathBuilder = new EsArchivePathBuilder(isServerless);
+  const auditbeatArchivePath = dataPathBuilder.getPath('auditbeat/hosts');
+
+  const { indexListOfDocuments } = dataGeneratorFactory({
+    es,
+    index: '.ml-anomalies-custom-v3_linux_anomalous_network_activity',
+    log,
+  });
+
+  const mlModuleName = 'security_linux_v3';
+  const mlJobId = 'v3_linux_anomalous_network_activity';
+  const baseRuleProps: MachineLearningRuleCreateProps = {
+    name: 'Test ML rule',
+    description: 'Test ML rule description',
+    risk_score: 50,
+    severity: 'critical',
+    type: 'machine_learning',
+    anomaly_threshold: 40,
+    machine_learning_job_id: mlJobId,
+    from: '1900-01-01T00:00:00.000Z',
+    rule_id: 'ml-rule-id',
+  };
+  let ruleProps: MachineLearningRuleCreateProps;
+  const baseAnomaly: Partial<Anomaly> = {
+    is_interim: false,
+    record_score: 43, // exceeds anomaly_threshold above
+    result_type: 'record',
+    job_id: mlJobId,
+    'user.name': ['root'],
+  };
+
+  // The tests described in this file rely on the 'alertSuppressionForMachineLearningRuleEnabled' feature flag
+  describe('@ess @serverless @serverlessQA Machine Learning Detection Rule - Alert Suppression', () => {
```

Comment:
This test, currently cannot be part of the `@serverlessQA` since on MKI we don't have support for feature flags yet. Moreover tests should be added once we know the tests is stable and is not failing on the periodic pipeline. 

Until the feature flag is not enabled by default, this test should be marked as `@skippedInServerlessMKI`

Reply:
[8e07763](https://github.com/elastic/kibana/pull/181926/commits/8e07763cee8c71b7fa389d1ad01ee7acea6db47a)